### ISSUE TITLE
Mypaint 2.0.0-1 loads no saved files - KeyError: u'file_summary'

### ISSUE THREAD

**Issue by OlliKn** [mypaint/mypaint#1061]
Recent saved files or with version 1.2 saved files can not be loaded. Not via the recent document dialog nor the open dialog. version 1.2 and krita can open the files.
while opening following error message appears. Ignoring it leads to the creation of a new empty document.

```python
Mypaint version: 2.0.0+git.af5225a
System information: Linux-5.3.0-40-generic-x86_64-with-debian-buster-sid
Using: Python 2.7.5, GTK 3.22.30, GdkPixbuf 2.36.12, Cairo 1.15.12, GLib 2.56.1
Traceback (most recent call last):
  File "/tmp/.mount_MyPainhIF9K7/usr/lib/mypaint/gui/filehandling.py", line 340, open_recent_cb(A=<gui.filehandling.FileHandler object>, action=<Gtk.RecentAction object at 0x7fdacfcc98c0 (GtkRecentAction at 0x2593100)>)
        if not D:return
        A.open_file(C)
      def open_last_cb(A,action):
  variables: {'C': ('local', u'/home/oliver/Bilder/ei.ora'), 'A.open_file': ('local', <bound method FileHandler.open_file of <gui.filehandling.FileHandler object at 0x7fdacfcce590>>)}
  File "/tmp/.mount_MyPainhIF9K7/usr/lib/mypaint/gui/filehandling.py", line 151, open_file(A=<gui.filehandling.FileHandler object>, filename=u'/home/oliver/Bilder/ei.ora', **F={})
        D=filename
        if not A._call_doc_load_method(A.doc.model.load,D,False,**F):A.filename=None;A.app.reset_compat_mode();A.doc.model.clear();return
        A.filename=os.path.abspath(D)
  variables: {'A._call_doc_load_method': ('local', <bound method FileHandler._call_doc_load_method of <gui.filehandling.FileHandler object at 0x7fdacfcce590>>), 'None': ('builtin', None), 'False': ('builtin', False), 'D': ('local', u'/home/oliver/Bilder/ei.ora'), 'F': ('local', {}), 'A.filename': ('local', <gui.filehandling.FileHandler object at 0x7fdacfcce590>), 'A.app.reset_compat_mode': ('local', <bound method Application.reset_compat_mode of <gui.application.Application object at 0x7fdad0cb1cd0>>), 'A.doc.model.clear': ('local', <bound method Document.clear of <Document nlayers=1 bbox=Rect(0, 0, 0, 0) paintonly=False>>), 'A.doc.model.load': ('local', <bound method Document.load of <Document nlayers=1 bbox=Rect(0, 0, 0, 0) paintonly=False>>)}
  File "/tmp/.mount_MyPainhIF9K7/usr/lib/mypaint/gui/filehandling.py", line 167, _call_doc_load_method(A=<gui.filehandling.FileHandler object>, method=<bound method Document.load of <Document nlayers=1 bbox=Rect(0, 0, 0, 0) paintonly=False>>, arg=u'/home/oliver/Bilder/ei.ora', is_import=False, compat_handler=None)
        if not B:B=compat.ora_compat_handler(A.app)
        D=A.app.preferences;E=D['display.colorspace'];F=is_import and'import'or'load';G=_IOProgressUI.format_files_summary(arg);C=_IOProgressUI(A.app,F,G);H=C.call(method,arg,convert_to_srgb=E=='srgb',compat_handler=B,incompatible_ora_cb=compat.incompatible_ora_cb(A.app));return H is not False and C.success
      def open_scratchpad(A,filename):
  variables: {'compat.incompatible_ora_cb': ('global', <function incompatible_ora_cb at 0x7fdad0e411b8>), '_IOProgressUI': ('global', <class gui.filehandling._IOProgressUI at 0x7fdad0bfc328>), 'C': (None, []), 'A.app.preferences': ('local', ObservableDict({'frame.color_rgba': [0.12, 0.12, 0.12, 0.92], 'ui.feedback.scale': False, u'colors.palette': {'name': u'MyPaint Default', 'columns': 7, 'entries': [('#000000', u'Black'), ('#262626', u'Gray 15'), ('#4d4d4d', u'Grey 30'), ('#808080', u'Grey 50'), ('#ababab', u'Grey 67'), ('#d9d9d9', u'Grey 85'), ('#ffffff', u'White'), ('#800000', u'Maroon'), ('#d10000', u'English-red'), ('#ff3622', u'Tomato'), ('#fc3c03', u'Orange-red'), ('#ff8c69', u'Salmon'), ('#fce820', u'Cadium-yellow'), ('#f3fd25', u'Lemon yellow'), ('#790523', u'Dark crimson'), ('#a9203e', u'Deep carmine'), ('#ff8c00', u'Orange'), ('#ffa812', u'Dark tangerine'), ('#d99058', u'Persian orange'), ('#c2b280', u'Sand'), ('#ffe5b4', u'Peach'), ('#64362e', u'Bole'), ('#6c2934', u'Dark cordovan'), ('#a3412c', u'Chestnut'), ('#e48864', u'Dark salmon'), ('#ffc38f', u'Apricot'), ('#ffdbbc', u'Unbleached silk'), ('#f2e3c6', u'Straw'), ('#35130d', u'Bistre'), ('#542a0e', u'Dark chocolate'), ('#66332b', u'Burnt sienna'), ('#b84200', u'Sienna'), ('#d8990c', u'Yellow ochre'), ('#d2b48c', u'Tan'), ('#e8cc90', u'Dark wheat'), ('#003153', u'Prussian blue'), ('#304577', u'Dark grey blue'), ('#0047ab', u'Cobalt blue'), ('#1f75fe', u'Blue'), ('#78b4ff', u'Bright french blue'), ('#abc8ff', u'Bright steel blue'), ('#d0e7ff', u'Ice blue'), ('#1e333a', u'Medium jungle green'), ('#2f4f4f', u'Dark slate grey'), ('#4a685d', u'Dark grullo green'), ('#008080', u'Teal'), ('#43aab0', u'Turquoise'), ('#6daec7', u'Cerulean frost'), ('#add9ba', u'Tiffany green'), ('#16221d', u'Gray-asparagus'), ('#24302d', u'Medium dark teal'), ('#4a685d', u'Xanadu'), ('#77c679', u'Mint'), ('#afcdb6', u'Timberwolf'), ('#b9f5f6', u'Celeste'), ('#c1ffea', u'Aquamarine'), ('#1d3423', u'Cal Poly Pomona'), ('#014421', u'Forest green'), ('#2a8000', u'Napier green'), ('#808000', u'Olive'), ('#419c69', u'Sea green'), ('#bdf61d', u'Green-yellow'), ('#e7f486', u'Bright chartreuse'), ('#8a1789', u'Purple'), ('#4e278a', u'Violet'), ('#c14b6e', u'Dark thulian pink'), ('#de3163', u'Cerise'), ('#ff1493', u'Deep pink'), ('#ff66cc', u'Rose pink'), ('#ffcbdb', u'Pink'), ('#ffffff', u'White'), ('#e51101', u'RGB Red'), ('#00ff00', u'RGB Green'), ('#0000ff', u'RGB Blue'), ('#00ffff', u'CMYK Cyan'), ('#ff00ff', u'CMYK Magenta'), ('#ffff00', u'CMYK Yellow')]}, u'color_chooser.fast_subset.window_size': [256, 264], 'ui.feedback.last_pos': False, u'colors.current': '#fffbfa', u'ui.blink_layers': True, 'view.default_zoom': 1.0, 'brushmanager.selected_groups': [], 'ui.toolbar_items': {u'toolbar1_file': True, u'toolbar1_view_modes': True, u'toolbar1_edit': True, u'toolbar1_blendmodes': False, u'toolbar1_view_manips': False, u'toolbar1_view_resets': True, u'toolbar1_scrap': False, u'toolbar1_linemodes': True}, 'default_compatibility_mode': '2.x', u'brush_chooser.default.window_size': [256, 306], 'input.button_mapping': {'<Primary>Button1': u'ColorPickMode', '<Primary>Button2': u'ZoomViewMode', '<Primary><Shift>Button2': u'FrameEditMode', '<Shift>Button1': u'StraightMode', '<Shift>Button2': u'RotateViewMode', 'Button3': u'ShowPopupMenu', 'Button2': u'PanViewMode', '<Shift><Alt>Button2': u'FrameEditMode', '<Alt>Button2': u'ZoomViewMode', '<Alt>Button1': u'ColorPickMode'}, 'ui.hide_toolbar_in_fullscreen': True, 'ui.dark_theme_variant': True, 'misc.context_restores_color': True, u'color_chooser.default.window_size': [256, 264], 'ui.parts': {u'main_toolbar': True, u'menubar': True}, 'ui.toolbar_icon_size': u'large', 'document.autosave_backups': True, 'display.colorspace': u'srgb', 'input.barrel_rotation_offset': 0.5, 'ui.rendered_tile_cache_size': 16384, 'ui.hide_subwindows_in_fullscreen': True, u'ui.file_compat_warning_mild': True, 'display.colorspace_EOTF': 2.2, 'input.use_barrel_rotation': True, u'colors.history': ['#003153', '#808080', '#ff3b2d', '#8e0900', '#fffbfa'], 'saving.scrap_prefix': u'~/MyPaint/scrap', 'ui.hide_menubar_in_fullscreen': True, u'ui.file_compat_warning_severe': True, u'brush_chooser.default.selected_group': u'set#2', 'input.barrel_rotation_subtract_ascension': True, 'input.global_pressure_mapping': [[0.0, 1.0], [1.0, 0.0]], 'workspace.layout': {u'fullscreen': False, u'right_sidebar': {u'h': 911, u'groups': [{u'tools': [], u'active_page': -1}], u'w': 8}, u'autohide': True, u'maximized': True, u'position': {}, u'floating': [], u'left_sidebar': {u'h': 911, u'groups': [{u'h': 434, u'tools': [[u'MyPaintModeOptionsTool']], u'w': 297, u'active_page': 0}, {u'h': 412, u'tools': [[u'MyPaintLayersTool']], u'w': 297, u'active_page': 0}, {u'tools': [], u'active_page': -1}], u'w': 297}}, 'compat_behavior_when_unknown': u'unless-pigment-layer-1.x', 'saving.default_format': u'openraster', 'brushmanager.selected_brush': u'ramon/Glow_Airbrush', u'ui.support_smooth_scrolling': True, u'input.devices': {u'Virtual core XTEST pointer:mouse:2': {}, u'AlpsPS/2 ALPS DualPoint TouchPad:mouse:4': {}, u'Logitech Optical USB Mouse:mouse:4': {}, u'AlpsPS/2 ALPS DualPoint Stick:trackpoint:4': {}, u'Wacom BambooFun 4x5 Pad pad:tablet-pad:6': {}, u'Wacom BambooFun 4x5 Pen stylus:pen:6': {}, u'Wacom BambooFun 4x5 Pen eraser:eraser:6': {}, u'AlpsPS/2 ALPS DualPoint TouchPad:touchpad:4': {}, u'Wacom BambooFun 4x5 Pen cursor:cursor:6': {}}, 'view.real_alpha_checks': True, 'input.device_mode': u'screen', 'document.autosave_interval': 10, 'scratchpad.last_opened_scratchpad': u'/home/oliver/.local/share/mypaint/scratchpads/autosave.ora', 'compability_settings': {u'2.x': {u'pigment_on_by_default': True, u'pigment_layer_is_default': True}, u'1.x': {u'pigment_on_by_default': False, u'pigment_layer_is_default': False}}})), 'E': ('local', u'srgb'), 'D': ('local', ObservableDict({'frame.color_rgba': [0.12, 0.12, 0.12, 0.92], 'ui.feedback.scale': False, u'colors.palette': {'name': u'MyPaint Default', 'columns': 7, 'entries': [('#000000', u'Black'), ('#262626', u'Gray 15'), ('#4d4d4d', u'Grey 30'), ('#808080', u'Grey 50'), ('#ababab', u'Grey 67'), ('#d9d9d9', u'Grey 85'), ('#ffffff', u'White'), ('#800000', u'Maroon'), ('#d10000', u'English-red'), ('#ff3622', u'Tomato'), ('#fc3c03', u'Orange-red'), ('#ff8c69', u'Salmon'), ('#fce820', u'Cadium-yellow'), ('#f3fd25', u'Lemon yellow'), ('#790523', u'Dark crimson'), ('#a9203e', u'Deep carmine'), ('#ff8c00', u'Orange'), ('#ffa812', u'Dark tangerine'), ('#d99058', u'Persian orange'), ('#c2b280', u'Sand'), ('#ffe5b4', u'Peach'), ('#64362e', u'Bole'), ('#6c2934', u'Dark cordovan'), ('#a3412c', u'Chestnut'), ('#e48864', u'Dark salmon'), ('#ffc38f', u'Apricot'), ('#ffdbbc', u'Unbleached silk'), ('#f2e3c6', u'Straw'), ('#35130d', u'Bistre'), ('#542a0e', u'Dark chocolate'), ('#66332b', u'Burnt sienna'), ('#b84200', u'Sienna'), ('#d8990c', u'Yellow ochre'), ('#d2b48c', u'Tan'), ('#e8cc90', u'Dark wheat'), ('#003153', u'Prussian blue'), ('#304577', u'Dark grey blue'), ('#0047ab', u'Cobalt blue'), ('#1f75fe', u'Blue'), ('#78b4ff', u'Bright french blue'), ('#abc8ff', u'Bright steel blue'), ('#d0e7ff', u'Ice blue'), ('#1e333a', u'Medium jungle green'), ('#2f4f4f', u'Dark slate grey'), ('#4a685d', u'Dark grullo green'), ('#008080', u'Teal'), ('#43aab0', u'Turquoise'), ('#6daec7', u'Cerulean frost'), ('#add9ba', u'Tiffany green'), ('#16221d', u'Gray-asparagus'), ('#24302d', u'Medium dark teal'), ('#4a685d', u'Xanadu'), ('#77c679', u'Mint'), ('#afcdb6', u'Timberwolf'), ('#b9f5f6', u'Celeste'), ('#c1ffea', u'Aquamarine'), ('#1d3423', u'Cal Poly Pomona'), ('#014421', u'Forest green'), ('#2a8000', u'Napier green'), ('#808000', u'Olive'), ('#419c69', u'Sea green'), ('#bdf61d', u'Green-yellow'), ('#e7f486', u'Bright chartreuse'), ('#8a1789', u'Purple'), ('#4e278a', u'Violet'), ('#c14b6e', u'Dark thulian pink'), ('#de3163', u'Cerise'), ('#ff1493', u'Deep pink'), ('#ff66cc', u'Rose pink'), ('#ffcbdb', u'Pink'), ('#ffffff', u'White'), ('#e51101', u'RGB Red'), ('#00ff00', u'RGB Green'), ('#0000ff', u'RGB Blue'), ('#00ffff', u'CMYK Cyan'), ('#ff00ff', u'CMYK Magenta'), ('#ffff00', u'CMYK Yellow')]}, u'color_chooser.fast_subset.window_size': [256, 264], 'ui.feedback.last_pos': False, u'colors.current': '#fffbfa', u'ui.blink_layers': True, 'view.default_zoom': 1.0, 'brushmanager.selected_groups': [], 'ui.toolbar_items': {u'toolbar1_file': True, u'toolbar1_view_modes': True, u'toolbar1_edit': True, u'toolbar1_blendmodes': False, u'toolbar1_view_manips': False, u'toolbar1_view_resets': True, u'toolbar1_scrap': False, u'toolbar1_linemodes': True}, 'default_compatibility_mode': '2.x', u'brush_chooser.default.window_size': [256, 306], 'input.button_mapping': {'<Primary>Button1': u'ColorPickMode', '<Primary>Button2': u'ZoomViewMode', '<Primary><Shift>Button2': u'FrameEditMode', '<Shift>Button1': u'StraightMode', '<Shift>Button2': u'RotateViewMode', 'Button3': u'ShowPopupMenu', 'Button2': u'PanViewMode', '<Shift><Alt>Button2': u'FrameEditMode', '<Alt>Button2': u'ZoomViewMode', '<Alt>Button1': u'ColorPickMode'}, 'ui.hide_toolbar_in_fullscreen': True, 'ui.dark_theme_variant': True, 'misc.context_restores_color': True, u'color_chooser.default.window_size': [256, 264], 'ui.parts': {u'main_toolbar': True, u'menubar': True}, 'ui.toolbar_icon_size': u'large', 'document.autosave_backups': True, 'display.colorspace': u'srgb', 'input.barrel_rotation_offset': 0.5, 'ui.rendered_tile_cache_size': 16384, 'ui.hide_subwindows_in_fullscreen': True, u'ui.file_compat_warning_mild': True, 'display.colorspace_EOTF': 2.2, 'input.use_barrel_rotation': True, u'colors.history': ['#003153', '#808080', '#ff3b2d', '#8e0900', '#fffbfa'], 'saving.scrap_prefix': u'~/MyPaint/scrap', 'ui.hide_menubar_in_fullscreen': True, u'ui.file_compat_warning_severe': True, u'brush_chooser.default.selected_group': u'set#2', 'input.barrel_rotation_subtract_ascension': True, 'input.global_pressure_mapping': [[0.0, 1.0], [1.0, 0.0]], 'workspace.layout': {u'fullscreen': False, u'right_sidebar': {u'h': 911, u'groups': [{u'tools': [], u'active_page': -1}], u'w': 8}, u'autohide': True, u'maximized': True, u'position': {}, u'floating': [], u'left_sidebar': {u'h': 911, u'groups': [{u'h': 434, u'tools': [[u'MyPaintModeOptionsTool']], u'w': 297, u'active_page': 0}, {u'h': 412, u'tools': [[u'MyPaintLayersTool']], u'w': 297, u'active_page': 0}, {u'tools': [], u'active_page': -1}], u'w': 297}}, 'compat_behavior_when_unknown': u'unless-pigment-layer-1.x', 'saving.default_format': u'openraster', 'brushmanager.selected_brush': u'ramon/Glow_Airbrush', u'ui.support_smooth_scrolling': True, u'input.devices': {u'Virtual core XTEST pointer:mouse:2': {}, u'AlpsPS/2 ALPS DualPoint TouchPad:mouse:4': {}, u'Logitech Optical USB Mouse:mouse:4': {}, u'AlpsPS/2 ALPS DualPoint Stick:trackpoint:4': {}, u'Wacom BambooFun 4x5 Pad pad:tablet-pad:6': {}, u'Wacom BambooFun 4x5 Pen stylus:pen:6': {}, u'Wacom BambooFun 4x5 Pen eraser:eraser:6': {}, u'AlpsPS/2 ALPS DualPoint TouchPad:touchpad:4': {}, u'Wacom BambooFun 4x5 Pen cursor:cursor:6': {}}, 'view.real_alpha_checks': True, 'input.device_mode': u'screen', 'document.autosave_interval': 10, 'scratchpad.last_opened_scratchpad': u'/home/oliver/.local/share/mypaint/scratchpads/autosave.ora', 'compability_settings': {u'2.x': {u'pigment_on_by_default': True, u'pigment_layer_is_default': True}, u'1.x': {u'pigment_on_by_default': False, u'pigment_layer_is_default': False}}})), 'G': ('local', u'\u201eei.ora"'), 'F': ('local', 'load'), 'H': (None, []), 'convert_to_srgb': (None, []), 'incompatible_ora_cb': (None, []), 'A.app': ('local', <gui.application.Application object at 0x7fdad0cb1cd0>), 'compat_handler': ('local', None), 'arg': ('local', u'/home/oliver/Bilder/ei.ora'), '_IOProgressUI.format_files_summary': ('global', <function format_files_summary at 0x7fdad0be5d70>), 'False': ('builtin', False), 'B': ('local', <function A at 0x7fdacc387410>), 'method': ('local', <bound method Document.load of <Document nlayers=1 bbox=Rect(0, 0, 0, 0) paintonly=False>>), 'is_import': ('local', False)}
  File "/tmp/.mount_MyPainhIF9K7/usr/lib/mypaint/gui/filehandling.py", line 43, __init__(A=<gui.filehandling._IOProgressUI instance>, app=<gui.application.Application object>, op_type='load', files_summary=u'\u201eei.ora"', use_statusbar=True, use_dialogs=True)
        if B not in A._OP_DURATION_TEMPLATES:raise ValueError('Unknown operation type %r'%(B,))
        C=A._OP_DURATION_TEMPLATES[B].format(files_summary=D);A._duration_msg=C;C=A._OP_SUCCEEDED_TEMPLATES[B].format(files_summary=D);A._success_msg=C;C=A._OP_FAILED_TEMPLATES[B].format(files_summary=D);A._fail_msg=C;C=A._OP_FAIL_DIALOG_TITLES[B];A._fail_dialog_title=C;A._is_write=B in['save','export'];E=A._app.statusbar.get_context_id('filehandling-message');A._statusbar_context_id=E;A._use_statusbar=bool(use_statusbar);A._use_dialogs=bool(use_dialogs);A.success=False;A._progress_dialog=None;A._progress_bar=None;A._start_time=None;A._last_pulse=None
      @with_wait_cursor
  variables: {'C': (None, []), 'B': ('local', 'load'), 'D': ('local', u'\u201eei.ora"'), 'format': ('builtin', <built-in function format>), 'files_summary': ('local', u'\u201eei.ora"'), 'A._OP_DURATION_TEMPLATES': ('local', {'load': u'\u201e{file_summary}\u201c wird geladen \u2026', 'import': u'Importiere Ebenen aus \u201e{files_summary}"\u2026', 'save': u'\u201e{files_summary}\u201c wird gespeichert \u2026', 'export': u'Export nach \u201e{files_summary}\u201c l\xe4uft \u2026'})}
KeyError: u'file_summary'
```

**Comment by jplloyd** [comment]
Thanks for reporting this.

Problem is caused by an incorrect translation (German language only). Temporary "solution" is to switch to English  or some other language (language options are in the View tab in the preferences) until this has been patched.

I'll write some tests tonight to see if any other languages have key problems, and then fix all of them at once (note to self: add this to the CI).

_(edited your comment for the missing backtick, fortunately there was nothing that was interpreted as an issue reference)_

**Comment by OlliKn** [comment]
That works for me. Thanks a lot for the quick solution! 
Will this issue closed by you?

**Comment by jplloyd** [comment]
Yes, I will close it once I've resolved it in the v2.0.x branch.

**Comment by jplloyd** [comment]
Key errors fix patch: [a553576b918f92dea8f8ea196d8561932fee7874](https://github.com/mypaint/mypaint/commit/a553576b918f92dea8f8ea196d8561932fee7874.patch)

Affected languages : Czech, German, Icelandic, Polish, Portuguese
